store/postgres/src/relational_queries.rs

```diff
@@ -818,9 +820,13 @@ impl<'a> QueryFilter<'a> {
             | In(attr, _)
             | NotIn(attr, _)
             | StartsWith(attr, _)
+            | StartsWithInsensitive(attr, _)
             | NotStartsWith(attr, _)
+            | NotStartsWithInsensitive(attr, _)
             | EndsWith(attr, _)
-            | NotEndsWith(attr, _) => {
+            | EndsWithInsensitive(attr, _)
+            | NotEndsWith(attr, _)
+            | NotEndsWithInsensitive(attr, _) => {
                 table.column_for_field(attr)?;
```

Comment:
LGTM but it might be a sign that we could use a `EntityFilter::attr(&self) -> Option<Attribute>` getter method for conciseness.  

Maybe we could open a new issue for this.